src/pages/Vehicle/InfoHeader/Label.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { colors, Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+
+interface IProps {
+  label: string;
+  title?: string;
+  type?: 'text' | 'progress';
+  progress?: number;
+}
+
+const Progress: React.FC<{ progress: number }> = ({ progress }) => {
```

Comment:
do you think component can be separated out to `components` folder ? this can be a reusable component as i had seen this in lot of other pages.

Reply:
yes we can